src/styles/blocks/cal.scss

```diff
@@ -0,0 +1,46 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * $cell-size;
```

Comment:
for `width` you should consider `cell-gap` and `calendar padding`

Reply:
I couldnt figure this one out either since I didnt use any padding for the calendar. Should I?